.travis.yml

```diff
@@ -91,8 +93,10 @@ matrix:
         apt:
           sources:
             - ubuntu-toolchain-r-test
+            - llvm-toolchain-trusty-3.9
           packages:
             - g++-6
+            - llvm-3.9-dev=1:3.9.1*
```

Comment:
Locally I find that yeah, the Ubuntu 14.04 (Trust) package doesn't seem to have 3.9.1 versioned the same as 
Ubuntu 16.04 (Xenial) does:
```
kevin@ubu-pony-2:~$ tail -4 /etc/apt/sources.list
#deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-3.9 main
#deb-src http://apt.llvm.org/xenial/ llvm-toolchain-xenial-3.9 main
deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-3.9 main
deb-src http://apt.llvm.org/trusty/ llvm-toolchain-trusty-3.9 main
kevin@ubu-pony-2:~$ 
kevin@ubu-pony-2:~$ 
kevin@ubu-pony-2:~$ sudo apt-get update
Hit:1 http://us.archive.ubuntu.com/ubuntu xenial InRelease
Get:2 http://apt.llvm.org/trusty llvm-toolchain-trusty-3.9 InRelease [4,222 B]                   
Get:3 http://us.archive.ubuntu.com/ubuntu xenial-updates InRelease [102 kB]                      
Get:4 http://security.ubuntu.com/ubuntu xenial-security InRelease [102 kB]          
Get:5 http://apt.llvm.org/trusty llvm-toolchain-trusty-3.9/main Sources [1,911 B]              
Get:6 http://apt.llvm.org/trusty llvm-toolchain-trusty-3.9/main amd64 Packages [6,625 B]         
Get:7 http://apt.llvm.org/trusty llvm-toolchain-trusty-3.9/main i386 Packages [6,652 B]   
Get:8 http://us.archive.ubuntu.com/ubuntu xenial-backports InRelease [102 kB]                    
Fetched 326 kB in 0s (375 kB/s)                                     
Reading package lists... Done
kevin@ubu-pony-2:~$ 
kevin@ubu-pony-2:~$ 
kevin@ubu-pony-2:~$ apt show llvm-3.9-dev
Package: llvm-3.9-dev
Version: 1:3.9~svn288847-1~exp1
Priority: optional
Section: devel
Source: llvm-toolchain-3.9
Maintainer: LLVM Packaging Team <pkg-llvm-team@lists.alioth.debian.org>
Installed-Size: 120 MB
Depends: libc6 (>= 2.14), libjsoncpp0 (>= 0.6.0~rc2), libllvm3.9 (= 1:3.9~svn288847-1~exp1), libstdc++6 (>= 4.7), libffi-dev (>= 3.0.9), llvm-3.9 (= 1:3.9~svn288847-1~exp1), libtinfo-dev
Replaces: llvm (<< 2.2-3)
Homepage: http://www.llvm.org/
Download-Size: 16.8 MB
APT-Sources: http://apt.llvm.org/trusty llvm-toolchain-trusty-3.9/main amd64 Packages
Description: Modular compiler and toolchain technologies, libraries and headers
...
```

So I presume that `llvm-3.9-dev=1:3.9*` would work here...